roles/istio/defaults/main.yaml

```diff
@@ -5,6 +5,5 @@ istio_create_seldon_gateway: true
 istio_ssl:
   enabled: true
   self_signed_ca: true
-#  server_crt_path: /path/to/server.crt
-#  server_key_path: /path/to/server.key
-
+  server_crt_path: null
```

Comment:
This way more clear that it can be provided

Reply:
I had originally tried out `default(omit)`

Reply:
unfortunately that doesn't work in this situation